### DISCUSSION TITLE
C7A bug?

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1891] [General]
I tried to add QSO with C7A, both ways (live QSO and post QSO) were unsuccessful.  Pressing the Save button does nothing.
The only way I found is to add QSO with another call (I added 3C7A) and edit that QSO afterwards.

**Comment by phl0** [comment]
Seems like Cloudlog cannot look up the DXCC for C7A. Select the DXCC in the "General" tab and it would save. The qrz.com page says: The DXCC Status not yet defined by DXAC / ARRL...

**Comment by magicbug** [comment]
Isn't a bug the callsign isn't assigned a DXCC, you can manually select Austria but wont be doing it automatically yet.

**Reply by R1BLH** [reply]
> Isn't a bug the callsign isn't assigned a DXCC, you can manually select Austria but wont be doing it automatically yet.

@phl0 

You are correct.  

I tried to add a QSO now and on the General tab I get the message "Please select element from list" in the DXCC field.
After selecting DXCC (None), I got the same message for the CQ Zone field,

After selecting the values for both fields QSO was added normally.

Would it be nice to have a predefined value of "Unknown" for these fields? 
In realtime work, the user could get confused (like me). 

@magicbug 

P.S. Happy New Year!

**Reply by phl0** [reply]
Or maybe allow the QSO to be logged without DXCC and ZONE info?

**Reply by magicbug** [reply]
If DXCC isn't found it should really select "None" which is in the list

**Reply by phl0** [reply]
> If DXCC isn't found it should really select "None" which is in the list

Ah that makes sense.
